sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesOperation.cs

```diff
@@ -148,6 +141,7 @@ internal AnalyzeHealthcareEntitiesOperation(TextAnalyticsRestClient serviceClien
             _diagnostics = diagnostics;
             _idToIndexMap = idToIndexMap;
             _showStats = showStats;
+            _operationInternal = new(nameof(AnalyzeHealthcareEntitiesOperation), _diagnostics, this);
```

Comment:
You can pull the type name off `this` parameter.

Reply:
I believe this won't work in some cases. For instance, in `KeyVault.Administration`, they have this `RestoreOperationInternal` class that contains the shared code between `RestoreOperation` and `SelectiveKeyRestoreOperation`. It's not a base class, and it's used internally (just like our `OperationInternal` is used). In this case, we would end up naming the scope as `RestoreOperationInternal.UpdateStatus`.

On the other hand, this feels like a specific scenario. Most LROs don't have this problem. I think it's worth pulling out the type name from `this`, as you suggested, and make an initializible property to override the default behavior if required.